Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsIgnoreHelpers.cs

```diff
@@ -4,7 +4,27 @@ namespace Leap.Unity.Interaction.PhysicsHands
 {
     // This script will prevent the physics hands helpers from being applied to your object.
     // This allows you to easily prevent important objects from being affected by the player.
-    // Note that this will not prevent your object from being collided with.
+
     public class PhysicsIgnoreHelpers : MonoBehaviour
-    { }
+    {
+        [Tooltip("This prevents the object from being collided with Physics Hands.")]
+        public bool DisableHandCollisions = true;
```

Comment:
Should this be false by default, so collisions can happen, but no helpers?

Reply:
I was going for true by default but happy to go either way.